requirements.txt

```diff
@@ -1,2 +1,3 @@
-PyYAML~=5.1
+PyYAML~=5.1; python_version > '3.4'
+PyYAML==3.11; python_version == '3.4'
```

Comment:
Are you sure these versions still work?

Reply:
yes, I pull a docker image to test with python 3.4.5 . and it's pass the CICD test